blocks/single-chain-block/chains/single-chain/default.test.jsx

```diff
@@ -14,8 +13,9 @@ describe('single chain', () => {
         </SingleChain>,
       );
 
-      expect(wrapper.text()).toBe(testText);
-      expect(wrapper.html()).toBe(`<p>${testText}</p>`);
+      expect(wrapper.html()).toMatchInlineSnapshot(
+        '"<div class=\\"sc-htpNat jHQBno\\"><p>one chainz</p></div>"',
```

Comment:
matching .html().children() or something close to that might be an improvement here. the classnames may change, right? I like the use of inline snapshot though

Reply:
The class name shouldn't change for this test. It would change based on the props used.